qpc/insights/publish.py

```diff
@@ -61,12 +62,63 @@ def _validate_insights_report_name(self):
             log.info(_(messages.INSIGHTS_LOCAL_REPORT_NOT_TAR_GZ))
             sys.exit(1)
 
+    def _validate_insights_report_content(self):
+        """Check if report tarball contains the expected json files."""
+        filenames = self._get_filenames()
+
+        if len(filenames) < 2:
+            log.error(_(messages.INSIGHTS_REPORT_CONTENT_MIN_NUMBER))
```

Comment:
Should all of these error messages related to validation failures be prefixed with something like `Payload file is not valid, due to: <specific error message>`?

Reply:
I think it's a good idea! I'm thinking on a adding this prefix directly on directly on the messages module (https://github.com/quipucords/qpc/blob/16f36451f4cdfa1f2e34f477e11b314b5d29edcd/qpc/messages.py#L362-L370).
I think I'd have to adapt a little bit the messages to accommodate the prefix :thinking:  
What do you think?

Reply:
(the linked snippet contains only the new messages)

Reply:
Adding the prefix directly to the messages is plenty good enough for this :+1

Reply:
done!